pycodestyle.py

```diff
@@ -2105,7 +2105,16 @@ def read_config(options, args, arglist, parser):
 
     parent = tail = args and os.path.abspath(os.path.commonprefix(args))
     while tail:
-        if config.read(os.path.join(parent, fn) for fn in PROJECT_CONFIG):
+        try:
+            files_read = config.read(os.path.join(parent, fn)
+                                     for fn in PROJECT_CONFIG)
+        except configparser.ParsingError:
+            if options.verbose:
+                print('failed to read local configuration files due to a '
+                      'parsing error')
```

Comment:
@ambv it's been a while since I worked on this. I'd look at the example config that was probably provided in the issue.